Dojo.Generators.Core/Dojo.Generators.Core.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<NuspecProperties>version=$(Version)</NuspecProperties>
```

Comment:
do you need to push it as separate nuget?

Reply:
Probably not.